### PR TITLE
Adds public and private character stats that can be used via events

### PR DESCRIPTION
**Pull request by Vassyli**
This PR updates dependencies and adds public and private character stats. Public character stats can be pulled for a "list of fighters", private ones for "current character stats", to be used during scene changes, for example.

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -51,8 +51,13 @@ swiftmailer:
 #graphql
 overblog_graphql:
     definitions:
-        internal_error_message: "An error occurred, please retry later or contact us!"
+        #internal_error_message: "An error occurred, please retry later or contact us!"
```
Comment (austenmc) [inline app/config/config.yml]: Can we not have comments?

File: app/config/routing.yml [hunk 2]
```diff
@@ -19,5 +19,5 @@ overblog_graphql_endpoint:
 
 #overblog_graphql_endpoint:
 #    resource: "@OverblogGraphQLBundle/Resources/config/routing/graphql.yml"
-overblog_graphql_graphiql:
-    resource: "@OverblogGraphQLBundle/Resources/config/routing/graphiql.yml"
\ No newline at end of file
+#overblog_graphql_graphiql:
```
Comment (austenmc) [inline app/config/routing.yml]: Do we really need comments here? We should just remove the code and keep it in the git history if we need it.

File: src/AppBundle/GraphQL/Types/CharacterType.php [hunk 3]
```diff
@@ -50,4 +61,101 @@ public function getDisplayName(): string
...
+        $stats = $this->returnBaseStats();
+
+        return $stats;
+    }
+
+    /**
+     * Returns public and private stats.
+     * @return array
+     */
+    public function getPrivateStats(): array
```
Comment (austenmc) [inline src/AppBundle/GraphQL/Types/CharacterType.php]: I dont like that getPrivateStats() returns both public and private. Either change it to getPublicAndPrivateStats or have it return only the private stats.

File: src/AppBundle/GraphQL/Types/CharacterType.php [hunk 4]
```diff
@@ -50,4 +61,101 @@ public function getDisplayName(): string
...
+    }
+
+
+    /**
+     * Returns base stats
+     * @return array
+     */
+    private function returnBaseStats(): array
+    {
+        $stats = [
```
Comment (austenmc) [inline src/AppBundle/GraphQL/Types/CharacterType.php]: nit: remove newline

**Review by austenmc — CHANGES_REQUESTED** [review]
In general this looks good, but Im not sure how useful the distinction between public and private stats are. Why not just have "stats" and then the system returns all the stats you are authorized to see?

Added some small nits and suggestions as well.

**Comment by Vassyli** [comment]
Yeah, that's a good suggestion. I'll look into changing it to "stats", which gets rows from `getPublicStats()` and, if the character is authorized, `getPrivateStats()`.

**Comment by Vassyli** [comment]
Done.

**Comment by austenmc** [comment]
Looks good. Let’s merge.

**Pull request merged**